### ISSUE TITLE
Addons installed

### ISSUE THREAD

**Issue by domnet73** [JamesLinEngineer/RKMC#19]
Hi, its possible one lite version of rkmc without addons ? Because i live in italy and  the addons installed is not for me......please use the default skin and addons of kodi..........thanks....good work

**Comment by superceleron** [comment]
Yes i think the addon's should not be there, only the default ones!
Just my 2 cents...

**Comment by domnet73** [comment]
Hi, I did not understand when I install the latest version of rkmc I find myself with a custom skin and with the addon in English of which I do not need that I can not even uninstall .......... It would be nice to have a clean version of rkmc ........ thanks

**Comment by domnet73** [comment]
hi, I did not understand, you are seeing to the removal of addons?

**Comment by mo123** [comment]
@domnet73 Root your device and delete the addons from RKMC's system's addon folder.

**Comment by domnet73** [comment]
Hi, my device is rooted.......but i cant delete addons installed in this version.......

**Comment by JamesLinEngineer** [comment]
@domnet73  @mo123 Hi, I will release a tiny RKMC version and use default skin and addons

**Comment by domnet73** [comment]
Thanks, I look forward to the lite version .......

**Comment by mo123** [comment]
@domnet73 
Here is a lite version of the latest RKMC without addons I made.
https://mega.nz/#!tMN0xD7C!NPQIEikDS47xCWPrY4kwA13pM9iBH1t6D0yH6az4rhA

**Comment by domnet73** [comment]
Thank you so much, I installed the "lite" version of rkmc is beautiful .... .... ....... lightweight and does not return errors. I hope you can continue working from a simple interface like this and without addon PREINSTALLED. Thank you

**Comment by mo123** [comment]
@domnet73 I see RKMC was updated again but the latest version has a bug where it stretches some videos fullscreen. When that bug is fixed then I will upload a lite version again.

**Comment by JamesLinEngineer** [comment]
@mo123  Hi, I fix this bug with the latest commit. please update and check it

**Comment by domnet73** [comment]
Hi, the l'asta version of rkmc have the addons installed.......and i don't disinstall this addons.....Why ? My Android is rooted !    Please resolve this problem because dropbox addons give me a error during update........one lite version clean and small its possible ! Thanks
